app/javascript/controllers/list_samples_controller.js

```diff
@@ -0,0 +1,41 @@
+import { Controller } from "@hotwired/stimulus";
+
+// creates a table listing all the selected samples
+export default class extends Controller {
+  static targets = ["field"];
+
+  #td_classes = ["bg-slate-50", "dark:bg-slate-900", "left-0", "left-12", "sticky"];
+
+  connect() {
+    const table = document.getElementById("samples-table");
+    const body = table.getElementsByTagName("tbody")[0];
+    const newTable = document.createElement("table");
```

Comment:
**suggestion:** This table will need table headers.  Try having the table already in the dialog with the headers and have the body as a target for this controller.